packages/block-library/src/paragraph/index.js

```diff
@@ -195,17 +194,14 @@ export const settings = {
 				return <RawHTML>{ attributes.content }</RawHTML>;
 			},
 			migrate( attributes ) {
-				return {
-					...attributes,
-					content: create( { html: attributes.content } ),
-				};
+				return attributes;
 			},
 		},
 	],
 
 	merge( attributes, attributesToMerge ) {
 		return {
-			content: concat( attributes.content, attributesToMerge.content ),
+			content: RichText.concat( attributes.content, attributesToMerge.content ),
```

Comment:
What's the difference between this and `attributes.content + attributesToMerge.content` ?

Will that difference be obvious to someone who thinks that they are working with strings?

Reply:
I'm actually changing this right now.

Reply:
Updated.